app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -287,26 +285,21 @@ public enum SettingsEnum {
     }
 
     private static void load() {
-        Context context = ReVancedUtils.getContext();
-        if (context == null) {
-            LogHelper.printException(() -> "SettingsEnum.load() called before ReVancedUtils.init()");
-            return;
-        }
         for (SettingsEnum setting : values()) {
             var path = setting.getPath();
             var defaultValue = setting.getDefaultValue();
             switch (setting.getReturnType()) {
                 case FLOAT:
-                    defaultValue = SharedPrefHelper.getFloat(setting.sharedPref, path, (float) defaultValue);
+                    defaultValue = SharedPrefHelper.getFloat(setting.sharedPref, path, (Float) defaultValue);
                     break;
                 case LONG:
-                    defaultValue = SharedPrefHelper.getLong(setting.sharedPref, path, (long) defaultValue);
+                    defaultValue = SharedPrefHelper.getLong(setting.sharedPref, path, (Long) defaultValue);
                     break;
                 case BOOLEAN:
-                    defaultValue = SharedPrefHelper.getBoolean(setting.sharedPref, path, (boolean) defaultValue);
+                    defaultValue = SharedPrefHelper.getBoolean(setting.sharedPref, path, (Boolean) defaultValue);
                     break;
                 case INTEGER:
-                    defaultValue = SharedPrefHelper.getInt(setting.sharedPref, path, (int) defaultValue);
+                    defaultValue = SharedPrefHelper.getInt(setting.sharedPref, path, (Integer) defaultValue);
```

Comment:
Why are objects used here instead of the primitives?

Reply:
Do primitives not implicitly work and shouldn't `SharedPrefHelper` methods use primitive types?